jekyll/_cci2/container-runner.adoc

```diff
@@ -95,6 +95,51 @@ agent:
           - name: "other"
 ```
 
+[#unsafe-retries]
+=== Unsafe retries
+
+Unsafe retries enable container runner to automatically rerun tasks if there are unexpected interruptions during their execution. These disruptions could be due to network connectivity issues, the underlying node shutting down, or other unpredictable causes. This feature can also be used when scheduling workloads on spot instances, which often come with cost-saving benefits at the risk of pod preemptions with many Kubernetes providers.
```

Comment:
Worth noting that errors surfaced in our UI as infrastructure failures should be expected to trigger an unsafe retry